ddtrace/debugging/_debugger.py

```diff
@@ -229,7 +230,7 @@ def __init__(self, tracer=None):
         )
 
         # Register the debugger with the RCM client.
-        RemoteConfig.register("LIVE_DEBUGGING", self.__rc_adapter__(self._on_configuration))
+        RemoteConfig.register(DEBUGGER_PRODUCT, self.__rc_adapter__(self._on_configuration))
```

Comment:
Perhaps we should make this change in a separate PR

Reply:
sounds good.